src/dispatch/static/dispatch/src/entity/EntityFilterCombobox.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <v-combobox
+    :items="items"
+    :label="label"
+    :loading="loading"
+    :search-input.sync="search"
+    @update:search-input="getFilteredData()"
+    chips
+    clearable
+    hide-selected
+    item-text="name"
+    item-value="id"
+    multiple
+    no-filter
+    v-model="entities"
+  >
+    <template v-slot:no-data>
+      <v-list-item>
+        <v-list-item-content>
+          <v-list-item-title>
+            No entities matching "
+            <strong>{{ search }}</strong
+            >"
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </template>
+    <template v-slot:selection="{ item, index }">
+      <v-chip close @click:close="entities.splice(index, 1)">
+        <span v-if="item.entity"> {{ item.name }} </span>{{ item.name }}
```

Comment:
```suggestion
        <span v-if="item.entity"> {{ item.value }} </span>{{ item.value }}
```

Reply:
Probably want the `entity_type.name` / `entity.value` so you can see the type and value of the entity and select